lib/src/chat/chat_page.dart

```diff
@@ -0,0 +1,298 @@
+import 'package:cloud_firestore/cloud_firestore.dart';
+import 'package:flutter/material.dart';
+import 'package:reazzon/src/helpers/spinner.dart';
+
+import 'chat_repository.dart';
+import 'message_model.dart';
+import 'message_page.dart';
+
+class ChatPage extends StatefulWidget {
+  @override
+  _ChatPageState createState() => _ChatPageState();
+}
+
+class _ChatPageState extends State<ChatPage> {
+  String UserID = 'OMA4VjyncrWeIlGIrGtVwLpLe3D3';
```

Comment:
@ujwalbasnet1 does this needs to be hardcoded?

Reply:
no this will be stored in sharedPreferences.